vignettes/matchprobes.Rmd

```diff
@@ -0,0 +1,201 @@
+---
+title: "Using oligonucleotide microarray reporter sequence information
+  for preprocessing and quality assessment"
+author: "H. Pagès, P. Aboyoun, R. Gentleman, and S. DebRoy"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+package: Biostrings
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+---
+
+# Overview
+
+This document presents some basic and simple tools for dealing with the oligonucleotide microarray reporter sequence information in the Bioconductor *probe* packages. This information is used, for example, in the *gcrma* package.
+
+*Probe* packages are a convenient way for distributing and storing the probe sequences (and related information) of a given chip.
+
+As an example, the package *hgu95av2probe* provides microarray reporter sequences for Affymetrix' *HgU95a version 2* genechip, and for almost all major Affymetrix genechips, the corresponding packages can be downloaded from the Bioconductor website. If you have the reporter sequence information of a particular chip, you can also create such a package yourself. This is described in in the makeProbePackage vignette of the *AnnotationForge* package.
+
+This document assumes some basic familiarity with R and with the design of the *AffyBatch* class in the *affy* package, Bioconductor's basic container for Affymetrix genechip data.
+
+First, let us load the *Biostrings* package and some other packages we will use.
+
+```{r include=FALSE}
+devtools::load_all(".")
+```
+
+```{r warning=FALSE, include=FALSE}
```

Comment:
Small edit: please change from r warning=FALSE, include=FALSE to r message=FALSE so that the libraries still show up once the file is knitted.